data/autoyast_xen/sles15sp1HVM_PRG.xml

```diff
@@ -169,7 +169,7 @@
       <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
       <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
       <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
-      <accept_verification_failed config:type="boolean">false</accept_verification_failed>
+      <accept_verification_failed config:type="boolean">true</accept_verification_failed>
```

Comment:
Are those really needed for 15-SP1 and others? For SLE 15 we know there is an issue until the installation medium gets updated, but for the others?

Reply:
I think so, but honestly I'm not 100% sure as it's already two days ago and I encountered many issues that day.
As 15-SP1 is also already in LTSS and we actually test the guest - not it's installation - I'd keep it here.

Reply:
I would then rather not do this for any flavor except SLE15 where this is a known issue.

If we get issues in the next runs, we know where they are coming from and fixing them is easy. But disabling certification verification by default reduced the coverage, especially in the guest installation step.

Reply:
Done. Please check.